### ISSUE TITLE
Problem with charts in bootstrap tab

### ISSUE THREAD

**Issue by PhilFlash** [krispo/angular-nvd3#85]
Hi,
I have problems with charts in bootstrap tab (angular 1.3.13, bootstrap 3.3.2)
All chart are very small in tab.

If I use the $scope.api functions in tab with a timeout, 
$scope.api in undefined in tab.
And it's not possible to use refresh() or update() to resize the chart.

Exemple here:
http://plnkr.co/edit/irJn2k?p=preview

**Comment by jsheedy** [comment]
I've been experiencing something similar.  One solution is to assign to $scope.data once the page is rendered.  I made 2 changes to your plunker and it works.  Updated example here:
http://plnkr.co/edit/wf32TpuzZXFPV9PWkH75

 I renamed $scope.data to $scope.data_src, then in your $viewContentLoaded handler assigned data_src to data:

                angular.extend($scope, {
                  data: $scope.data_src
                });

Something similar could be done with the pie charts.

**Comment by PhilFlash** [comment]
The solution is to use the dot notation for api.
https://github.com/angular/angular.js/wiki/Understanding-Scopes
*This issue with primitives can be easily avoided by following the "best practice" of always have a '.' in your ng-models.*
The solution:
```javascript
        $scope.api = {};
        $scope.select = function (arg) {
            $scope.selectedTab = arg;
            $timeout(function () {
                 if (arg === 1) $scope.api.chart1.update();
                 else if (arg === 2) $scope.api.chart2.update();
                 else if (arg === 3) $scope.api.chart3.update();
   
            });
        };
```
```html
<nvd3 options="options" data="data" api="api.chart1"></nvd3>
```
I have updated the plunker and all is ok.
http://plnkr.co/edit/irJn2k?p=preview

**Comment by krispo** [comment]
Hi,
development in master (and 1.x) branch relates to the recent nvd3 versions (v1.7.1+) and it's incompatible with the old one (v1.1.15-beta).

I've created 0.x branch to maintain the old nvd3 version (v1.1.15-beta). It will include all zero-based angular-nvd3 versions like (v0.0.x, v0.1.x,..).

So in the plunker example it should be used the latest stable angular-nvd3 version (v0.1.1), not master. I've modified a bit your example:
http://plnkr.co/edit/o6PFs8?p=preview

Thanks for the issue!
